source/engineering/OpenFOAM/multi-node.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Multi-Node Jobs"
+---
+
+Large OpenFOAM jobs can be run on multi nodes.
+
+
+## Job Script Template
+
+Here is an example job submission script for a multi-node OpenFOAM job:
+
+```{code-block} bash
+---
+linenos: true
+---
+#!/bin/bash
+## Submission script for Cluster
+#SBATCH --job-name=my_job
+#SBATCH --time=0-01:00:00 # hh:mm:ss
+#SBATCH --nodes=4
+#SBATCH --ntasks-per-node=28
+#SBATCH --cpus-per-task=1
+
+
+#SBATCH --partition=cpu
+
+## system error message output file
+## leave %j as it's being replaced by JOB ID number
+#SBATCH -e aai_LD_%j.err
+
+## system message output file
+#SBATCH -o aai_LD_%j.out
+
+# record some potentially useful details about the job: 
+echo Running on host $(hostname) 
+echo Time is $(date) 
+echo Directory is $(pwd) 
+echo Slurm job ID is ${SLURM_JOBID} 
+echo This jobs runs on the following machines: 
+echo ${SLURM_JOB_NODELIST} 
+printf "\n\n\n\n" 
+
+# Load modules required for runtime e.g.
+module load apps/openfoam/6
+
+#export I_MPI_PROCESS_MANAGER=mpd
+
+
+# Run the solver. Take pisoFoam with 112 processors for example:
+mpirun -np 112 pisoFoam -parallel 
+
+echo End Time is $(date) 
+echo "Done pimpleFoam finish"
+printf "\n\n"
+
+```
+
+```{note}
+Further explanation of the script could be found in: https://www.acrc.bris.ac.uk/protected/hpc-docs/job_types/index.html
+```
+### How to use
+Save the script file in the case folder, and submit it to Slurm after the decomposition of mesh and initial field data. The processors number is the nodes number times the number of tasks per node. 
```

Comment:
It might be useful to link to some documentation on the case folder.

```suggestion
Save the script file in the [case folder](https://www.openfoam.com/documentation/user-guide/2-openfoam-cases), and submit it to Slurm after the decomposition of mesh and initial field data. The number of MPI processes to request is the number of nodes times the number of tasks per node. 
```